reframe/core/buildsystems.py

```diff
@@ -246,31 +246,31 @@ def emit_build_commands(self, environ):
         fflags   = self._fflags(environ)
         ldflags  = self._ldflags(environ)
         if cc is not None:
```

Comment:
Maybe you can change all the `if <var> is not None:` with `if <var>:`

Reply:
No, it has a different meaning if it is `None`. Check [here](https://reframe-hpc.readthedocs.io/en/stable/reference.html#reframe.core.buildsystems.BuildSystem). In short if it is not `None` but empty, it will be emitted the following: `cc=''`. Same thing for the different flags.